### PR TITLE
Put in a fix to not render the Desktop and Mobile UCP players when th…

### PR DESCRIPTION
**Pull request by p-forfa**
…e window.canPlayVideo() function returns false; this includes guards in case the functions is undefined, or even if the window variable is not defined for whatever reason, including SSR

### CODE REVIEW

File: src/loaders/DesktopArticleVideoLoader.tsx [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
 import { setVersionWindowVar } from 'loaders/utils/GetVersion';
+import { shouldLoadUcpPlayer } from 'loaders/utils/shouldLoadPlayer';
 
 export { getVideoPlayerVersion } from 'loaders/utils/GetVersion';
 
 export const DesktopArticleVideoLoader: React.FC<DesktopArticleVideoLoaderProps> = ({ videoDetails }) => {
+	if (!shouldLoadUcpPlayer()) {
+		return;
+	}
+
```
Comment (AlburIvan) [inline src/loaders/DesktopArticleVideoLoader.tsx:12]: Quick question, can one do that? 

Won't it have an issue with `hooks` cannot be called conditionally ?

Comment (gundams-and-dragons) [reply src/loaders/DesktopArticleVideoLoader.tsx:12]: I think the hooks will get called either way, but I'd move this below those, just above the other returns

Comment (gundams-and-dragons) [reply src/loaders/DesktopArticleVideoLoader.tsx:12]: Actually inside the effect hook would probably be best, prevent it from getting run altogether

Comment (gundams-and-dragons) [reply src/loaders/DesktopArticleVideoLoader.tsx:12]: Maybe something like

if (!player && shouldLoadUcpPlayer()) {
	getPlayer();
}

Comment (p-forfa) [reply src/loaders/DesktopArticleVideoLoader.tsx:12]: I'll make the change. There's a deploy running on sandbox-s8 right now. I'll verify it works then, then redeploy with the above change you both discussed.

Comment (p-forfa) [reply src/loaders/DesktopArticleVideoLoader.tsx]: @gundams-and-dragons @AlburIvan Fixed in 4398dd619dc73b31e4c4b49a8e1ef58f2f0e5a85

**Review by gundams-and-dragons — APPROVED** [review]

**Review by AlburIvan — APPROVED** [review]

File: src/loaders/DesktopArticleVideoLoader.tsx (hunk 1 above)
Comment (JoshuaRogan) [reply src/loaders/DesktopArticleVideoLoader.tsx]: @gundams-and-dragons is `canPlayVideo` created from the php side too? If not the hook could be an issue if our code somehow runs first.

Comment (JoshuaRogan) [reply src/loaders/DesktopArticleVideoLoader.tsx]: Just checked warrens slack. LGTM

**Pull request merged**